src/components/Header/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import SearchBar from './index';
+
+test('renders SearchBar', () => {
```

Comment:
Always add a `describe` for your set of test cases, this will help you keep your tests organized as the tests cases grow. For now it could be as simple as:
```javascript
describe('SearchBar component', () => {
  test('renders SearchBar', () => {
    ...
  });
});
```